apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -67,7 +67,7 @@
 
   Quill.register('modules/clipboard', PotentiallyDisabledClipboard, true);
 
-  window.PLRTE = function (uuid) {
+  window.PLRTE = async function (uuid) {
```

Comment:
Nit, the Mustache template has something like this:

```js
var rte = window.PLRTE(...);
```

Maybe best to just remove the variable assignment?

I'm mildly uneasy about using async functions at the top level (either called in a `<script>` tag or in `$(() => { ... })`) mainly because of error handling, but in practice this shouldn't really cause problems even if there are errors. That uneasiness is probably just coming from experience with Node where we really don't want unhandled errors/rejections.

Reply:
Removed variable in 357a91d8c2ad9e1718e060607ff5eebddde91e5c. I also added a comment that we're deliberately calling an async function without await. An alternative to this model could be a selector-observer in the JS file itself that is called when the element is loaded into the DOM, but I think that may be unnecessary complexity.